test/common/pixel-tests

```diff
@@ -1,19 +1,14 @@
 #!/bin/bash
 
-set -eu
+GITHUB_REPO='pixel-test-reference'
+SUBDIR='test/reference'
 
-SUBDIR=test/reference
-REPO=pixel-test-reference
+set -eu
+cd "$(realpath -m "$0"/../../..)"
+. tools/git-utils.sh
```

Comment:
The `pixel-tests` tool is also used by external projects, and they only have `test/common`, not `tools`.  So this here will fail, no?

Reply:
I wasn't aware of that.  I'll drop this commit for the time being.